content/md/user-management/manage-your-users.md

```diff
@@ -63,7 +63,7 @@ Even if a user is deleted, its actions generated in the PIM won't be erased.
 
 # Deal with the user permissions
 
-To know more about user permissions, there are four articles just for you depending on what your need are: 
+To know more about user permissions, there are four articles just for you depending on what your need are:
```

Comment:
Starting this point you already have these changes in another PR. I can help you with Git to remove them from here.